akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -98,12 +102,12 @@ private[stream] object GraphInterpreter {
 
       for (i ← 0 until connectionCount) {
```

Comment:
yes, not for this test but we need materialization to be fast and ranges are extremely slow (see for example http://www.scalablescala.com/roller/scala/entry/why_is_using_for_foreach)